File: Back/src/Fullstack.Domain/Palestrante.cs
using Fullstack.Domain.Identity;

namespace Fullstack.Domain
{
    public class Palestrante
    {
    public int Id { get; set; }
    public string MiniCurriculo { get; set; }
    public int UserId { get; set; }
    public User User {get; set;}
    public IEnumerable<RedeSocial> RedeSociais { get; set; }
    public IEnumerable<PalestranteEvento> PalestranteEventos { get; set; }
}
}

File: Front/FullStack-App/src/app/models/Palestrante.ts
import { Evento } from './Evento';
import { UserUpdate } from './Identity/UserUpdate';
import { RedeSocial } from './RedeSocial';

export interface Palestrante {
  id: number;
  nome: string;
  miniCurriculo: string;
  user: UserUpdate;
  redesSociais: RedeSocial[];
  palestrantesEventos: Evento[];
}
